src/cli/build.helpers.ts

```diff
@@ -209,3 +209,24 @@ export async function generateYamlAndWriteToDisk(obj: {
   await assets.generateHelmChart(webhookConfigGenerator, outputDir);
   console.info(`✅ K8s resource for the module saved to ${yamlPath}`);
 }
+
+export interface BuildOpts {
+  customImage?: string;
+  registryInfo?: string;
+  registry?: "GitHub" | "Iron Bank";
+}
```

Comment:
Just so it doesn't get lost: we talked at standup about adding the --version flag into this set too, yes?